crates/bevy_image/src/image.rs

```diff
@@ -1054,6 +1054,60 @@ impl Image {
         offset.map(|start| &mut data[start..(start + len)])
     }
 
+    /// Returns an iterator over the image's pixels.
```

Comment:
column-row or row-column? starting from which corner?

Reply:
How does "The iteration order is x = 0 to width then y = 0 to height then z = 0 to depth." sound?